BLT/BLT/UserData.swift

```diff
@@ -56,6 +56,12 @@ class UserData: Codable {
 			saveUserData()
 		}
 	}
+    
+    var focusTime: Double = 0 {
```

Comment:
This should be calculated by the app dynamically it cannot be a variable that we just keep adding to.